syncd/Syncd.cpp

```diff
@@ -2612,6 +2612,16 @@ sai_status_t Syncd::processOidCreate(
 
             m_switches[switchVid] = std::make_shared<SaiSwitch>(switchVid, objectRid, m_client, m_translator, m_vendorSai);
 
+            /*
+             * If IPC server is enabled, it needs to remember switch object rid and
+             * switch api, used by the calling of switch_mdio_read/write.
+             */
+            if (m_ipcServer && m_switches.size() == 1)
```

Comment:
this will onlyu work for first switch and possible if many will be created then this will not be executed